packages/x-components/tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+const { xTailwindPlugin } = require('@empathyco/x-tailwindcss');
+
+module.exports = {
+  content: [
+    './index.html',
+    './src/components/**/*.vue',
```

Comment:
I don't think so, according to the [tailwind docs](https://tailwindcss.com/docs/configuration#content) we should add the paths to any file that could contain `Tailwind` class names and as far as I can see this is not the case for those files, is it? At least we are not doing that yet.

Reply:
We have some components, like animations built in `.ts` files instead of `.vue`, but they are not using tailwind classes yet. We can keep it like this then.